### DISCUSSION TITLE
When can preview 5 be released? The nuget package I made with source code has never been used!

### DISCUSSION THREAD

**Discussion by wj8400684** [AvaloniaUI/Avalonia#9920] [Q&A]
When can preview 5 be released? The nuget package I made with source code has never been used!

**Comment by timunie** [comment]
Soon it will be released. Probably next week already.

**Reply by wj8400684** [reply]
XamlX.XamlTypeSystemException: Unable to resolve type Avalonia.Controls.IControl
   in XamlX.TypeSystem.XamlTypeSystemExtensions.GetType(IXamlTypeSystem sys, String type)
   in Avalonia.Markup.Xaml.XamlIl.CompilerExtensions.AvaloniaXamlIlLanguage.Configure(IXamlTypeSystem typeSystem)
   in Avalonia.Build.Tasks.XamlCompilerTaskExecutor.CompileCore(IBuildEngine engine, CecilTypeSystem typeSystem, String projectDirectory, Boolean verifyIl, MessageImportance logImportance, Boolean debuggerLaunch)
   in Avalonia.Build.Tasks.XamlCompilerTaskExecutor.Compile(IBuildEngine engine, String input, String[] references, String projectDirectory, String output, Boolean verifyIl, MessageImportance logImportance, String strongNameKey, Boolean skipXamlCompilation, Boolean debuggerLaunch)
   in Avalonia.Build.Tasks.CompileAvaloniaXamlTask.Execute()
   in Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   in Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	AvaloniaApplication1	C:\Users\84006\.nuget\packages\avalonia\11.0.0-preview11\buildTransitive\AvaloniaBuildTasks.targets	96

**Reply by wj8400684** [reply]
How to solve it?

**Reply by wj8400684** [reply]
I downloaded the latest source code myself and made nuget. This error occurred

**Reply by timunie** [reply]
1. Make sure to read build instructions
2. Make sure to init submodules 
3. There's also a nightly feed, so you don't have to build it on your own if you're interested in latest master builds.

**Reply by maxkatz6** [reply]
Avalonia.Controls.IControl interface was removed. Now you need to use specific Control type. Same with many other interfaces.
If it's not your code, double check if you are using latest XamlNameReferenceGenerator package version as well.

**Reply by wj8400684** [reply]
Where can I get the XamlNameReferenceGenerator?

**Reply by maxkatz6** [reply]
https://www.nuget.org/packages/XamlNameReferenceGenerator

**Comment by JTtheGeek** [comment]
where can we find the changes we need to make from 4-5?  ViewLocator in existing sample is broken, along with FluentTheme it seems
